.readthedocs.yml

```diff
@@ -14,4 +14,5 @@ python:
   install:
     - method: pip
       path: .
-    - requirements: docs/requirements.txt
+      extra_requirements:
+        - docs
```

Comment:
@pllim If I understand what this line in the rtd.yml file is doing, it should be instructing RTD to look in setup.cfg for the `-docs` section of `[options.extras_require]` in order to find the documentation build requirements. This seems to be the way it works [on this line](https://github.com/astropy/astroquery/blob/436245056c3e624b08abb0095a1bbec642ed1526/.readthedocs.yaml#L10) of the .readthedocs.yml file in astroquery (this section of the astroquery setup.cfg is [here](https://github.com/astropy/astroquery/blob/436245056c3e624b08abb0095a1bbec642ed1526/setup.cfg#L172)). But numpy appears in this section of my setup.cfg, so I'm not sure why my RTD build has this persistent error `ModuleNotFoundError: No module named 'numpy'`